lib/dvb/pmt.cpp

```diff
@@ -653,18 +654,28 @@ int eDVBServicePMTHandler::getProgramInfo(program &program)
 				int x = 1;
 				for (std::vector<std::string>::iterator it = autoaudio_languages.begin();x <= autoaudio_level && it != autoaudio_languages.end();x++,it++)
 				{
-					if ((*it).find(program.audioStreams[i].language_code) != std::string::npos)
+					if ((*it).find(program.audioStreams[i].language_code.substr(0, program.audioStreams[i].language_code.find('/'))) != std::string::npos)
```

Comment:
this fixes when language code is transmitted as lang1/lang2.
It could also be fixed in lib/dvb/pmtparse.cpp on line 367, but I can't tell if it's a better solution.
If the line is too long, then the call to substr(0, prog...) can be shortened to substr(0,3), but I think that this tells the reader more about what is happening.